cloudflare.sh

```diff
@@ -63,30 +65,30 @@ test "$1" = "verify" && {
 
 idsfile=$(mktemp /tmp/cloudflare-ids-XXXXXX)
 zonefile=$(mktemp /tmp/cloudflare-zone-XXXX)
-cat > $zonefile
-trap "rm -f $idsfile $zonefile $zonefile-data" EXIT
+cat > "$zonefile"
+trap 'rm -f $idsfile $zonefile $zonefile-data' EXIT
```

Comment:
shouldn't it be ?
trap 'rm -f "$idsfile" "$zonefile" "$zonefile-data"' EXIT

Reply:
Yes. Thanks. Fixing now...